docs/release-notes/8.12.asciidoc

```diff
@@ -0,0 +1,107 @@
+[[release-notes-header-8.12.0]]
+== 8.12
+
+[discrete]
+[[release-notes-8.12.0]]
+=== 8.12.0
+
+[discrete]
+[[known-issue-8.12.0]]
+==== Known issues
+
+
+// tag::known-issue-173958[]
+[discrete]
+.[Title]
```

Comment:
Title is still a placeholder.

Reply:
@yctercero do you have any suggestions for the known issue title? Would this work?

```suggestion
.Data view option incorrectly displays when editing a filter applied to the KQL query bar 
```

Reply:
That looks great!